barretenberg/cpp/src/barretenberg/proof_system/debug_utility.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "barretenberg/common/log.hpp"
```

Comment:
nit: usually there's a better name than `X_utility`. But to be clear, it's holidays, get'er in

Reply:
changed to `instance_inspector` - useful for now but may delete in the near future